src/service.js

```diff
@@ -390,7 +412,7 @@ module.exports = function(mixinOptions) {
 						const resolver = {};
 
 						Object.values(service.actions).forEach(action => {
-							const { graphql: def } = action;
+							const { graphql: def, fileUploadArg } = action;
```

Comment:
Since this `fileUploadArg` is specific to GraphQL, it would probably be best to keep it located within the `graphql` property of the action schema rather than introducing a new action schema property.